torch/_inductor/cudagraph_trees.py

```diff
@@ -1083,7 +1111,18 @@ def __init__(self, device_index: int):
         # number of instances we had to checkpoint the function
         self.debug_checkpointing_counter = 0
 
+        self.id_to_mode: Dict[int, CompilationMode] = {}
+
+        # forwards that have been invoked without invocation of their corresponding backwards
+        self.called_forwards_on_stack: int = 0
```

Comment:
can we call this something more meaningful, like `forwards_with_pending_backwards`